target/linux/ath79/mikrotik/config-default

```diff
@@ -38,3 +38,4 @@ CONFIG_UBIFS_FS_ZLIB=y
 CONFIG_WATCHDOG_CORE=y
 CONFIG_ZLIB_DEFLATE=y
 CONFIG_ZLIB_INFLATE=y
+CONFIG_NET_DSA=y
```

Comment:
This should be put in the appropriate position, or preferably, be refreshed with make kernel_oldconfig TARGET=subtarget

Reply:
It does not seem to use DSA at all?

Reply:
> It does not seem to use DSA at all?

It doesn't, but DSA is needed so that SFP and PHYLINK modules are actually included. Otherwise, they are not added to the firmware image.

Reply:
> This should be put in the appropriate position, or preferably, be refreshed with make kernel_oldconfig TARGET=subtarget

Fixed (manually moved to the correct position).